vcl_templates/www.vcl.erb

```diff
@@ -303,6 +303,11 @@ sub vcl_recv {
     set req.url = querystring.remove(req.url);
   }
 
+  if (req.url.path ~ "^\/alerts(\/.*)?$") {
```

Comment:
simplifies to
```suggestion
  if (req.url.path ~ "^\/alerts\/?") {
```

(I double-checked that VCL's `~` is indeed a [partial match](https://docs.fastly.com/en/guides/vcl-regular-expression-cheat-sheet#matching-at-the-beginning-or-end-of-a-string) rather than a full match.)

Reply:
Lovely stuff, thanks 👍

Reply:
(updated the suggestion)

It's not quite as much of a simplification as I'd first thought, but it does at least save some computation and possibly some memory 🤷 

Some quick tests to check I've not messed up: https://regexr.com/7ca86